docs/specs/xref.yml

```diff
@@ -42,3 +42,65 @@ outputs:
     { "content": "<p>Link to <a href=\"https://docs.microsoft.com/en-us/dotnet/api/system.string\">String</a></p>\n"}
   build.manifest:
 ---
+# Define uid in conceptual, resolve it
+inputs:
+  docfx.yml: |
+    buildInternalXrefMap: true
```

Comment:
move this to `docfx.test.yml` ?

Reply:
It is default as false. Adding a TODO to remove it when we fix the performance.

---
In reply to: [219040625](https://github.com/dotnet/docfx/pull/3437#discussion_r219040625) [](ancestors = 219040625)